ImgSmall.java

```diff
@@ -0,0 +1,48 @@
+import javax.imageio.ImageIO;
+import java.awt.Image;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+
+public class ImgSmall {
+  public static void main(String[] args) throws IOException {
+      resize(args);
+  }
+  public static void resize(String[] args) throws IOException{
+    //get url object
+    URL url = null;
+    try{
+      url = new URL(args[0]);
+    } catch (Exception MalformedURLException){
+      System.out.println("check the url");
+      System.exit(0);
+    }
+    double resizeBy = (100-Integer.parseInt(args[1]))/100.0;
+
```

Comment:
Formatting issue. 
(100 - Integer.parseInt(args[1])) / 100.0; 
// not the spaces between 100 and - and Integer.